.tests/cypress/integration/log_in/registration/registration.js

```diff
@@ -0,0 +1,34 @@
+var faker = require('faker')
+
+describe('Registration test', () => {
+    before(() => {
+      cy.account()
+    })
+
+    it('checks New Customer header title is visible', () => {
```

Comment:
Those 2 "it" can be in one. We should not make one "it"  per checking one element